lepton/provider.go

```diff
@@ -36,7 +36,7 @@ type Provider interface {
 	CreateInstance(ctx *Context) error
 	ListInstances(ctx *Context) error
 	GetInstances(ctx *Context) ([]CloudInstance, error)
-	GetInstanceByID(ctx *Context, id string) (*CloudInstance, error)
+	GetInstance(ctx *Context, name string) (*CloudInstance, error)
```

Comment:
I've some concerns about the name of the function `GetInstance`. This name is too similar to `GetInstances`. Having a name like `GetInstancebyName` increases the code readability.

Reply:
Fixed by https://github.com/nanovms/ops/pull/1034/commits/99369f74d2f8b633ea1b35b374a12ed199e7cc12